services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from typing import Union
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> Union[MovieSession, QuerySet]:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+    return movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    movie_sessions = MovieSession.objects.all()
+
+    if session_date:
+        session_date = datetime.strptime(session_date, "%Y-%m-%d").date()
+        movie_sessions = movie_sessions.filter(show_time__date=session_date)
+
+    return movie_sessions
+
+
+def get_movie_session_by_id(movie_id: int) -> Union[MovieSession, QuerySet]:
+
+    return MovieSession.objects.get(id=movie_id)

```

Comment:
```suggestion
def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
    return MovieSession.objects.get(id movie_session_id)
```

get_movie_session_by_id, takes movie_session_id - id of the movie session, returns movie session with the provided id

![image](https://github.com/mate-academy/py-movie-moviesession-cinemahall/assets/93856423/77992de4-d353-4926-aff6-093c3a7a6275)